tests/conftest.py

```diff
@@ -147,7 +147,7 @@ def __init__(
         self,
         model_name: str,
         dtype: str = "half",
-        access_token: Optional[str] = None,
+        **kwargs,
```

Comment:
* access token not needed
* kwargs enables us to pass whatever we want for hf runner